src/ec/eddsa.rs

```diff
@@ -168,19 +212,16 @@ fn public_from_private(seed: &Seed, out: &mut PublicKey) {
     }
 }
 
-
 extern  {
     fn GFp_ed25519_scalar_mask(a: &mut [u8; SCALAR_LEN]);
-    fn GFp_ed25519_sign(out_sig: *mut u8/*[64]*/, message: *const u8,
-                        message_len: c::size_t, private_key: *const u8/*[64]*/);
-
     fn GFp_ed25519_verify(message: *const u8, message_len: c::size_t,
                           signature: *const u8/*[64]*/,
                           public_key: *const u8/*[32]*/) -> c::int;
-
     fn GFp_ge_p3_tobytes(s: &mut [u8; ELEM_LEN], h: &Point);
     fn GFp_x25519_ge_scalarmult_base(h: &mut Point, a: &Seed);
-
+    fn GFp_x25519_sc_muladd(s: *mut u8, a: *const u8, b: *const u8,
+                            c: *const u8);
```

Comment:
Should these parameters be of type `[u8; SCALAR_LEN]` and similar, instead of pointers?